### PR TITLE
Update for cuDF 22.08

### PR DESCRIPTION
**Pull request by shwina**

### CODE REVIEW

File: shareable-dataframes/python/kernel.cpp [hunk 1]
```diff
@@ -1,4 +1,56 @@
-/* Generated by Cython 0.29.23 */
+/* Generated by Cython 0.29.30 */
```
Comment (VibhuJawa) [inline shareable-dataframes/python/kernel.cpp:1]: I am not sure we should include an auto generated file in the PR anymore.  May-be cleaner if we just provide users with a command to do it.

**Review by VibhuJawa — CHANGES_REQUESTED** [review]
Looks good to me, have just requested changes around not including the auto generated file anymore.e

File: shareable-dataframes/python/kernel.cpp (hunk 1 above)
Comment (shwina) [reply shareable-dataframes/python/kernel.cpp:1]: I've gone ahead and done that in the README.

Comment (shwina) [reply shareable-dataframes/python/kernel.cpp]: I also nuked the autogenerated `.cpp` file

Comment (VibhuJawa) [reply shareable-dataframes/python/kernel.cpp]: Perfect , Thanks for the help with the cleanup Ashwin

**Review by VibhuJawa — APPROVED** [review]
LGTM to me. 


@jdye64 , In case you want to review this over too before merging.

**Comment by jdye64** [comment]
Thanks for the update and sorry for the delay

**Pull request merged**